doc/sphinx/lb.rst

```diff
@@ -121,53 +121,53 @@ load in the particles with the correct forces, and use::
 
     sys.integrator.run(steps=number_of_steps, reuse_forces=True)
 
-upon the first call to :ref:`run <Integrator>`. This causes the
+upon the first call ``integrator.run``. This causes the
 old forces to be reused and thus conserves momentum.
 
-.. _LB as a thermostat:
+.. _Interpolating velocities:
 
-LB as a thermostat
-------------------
+Interpolating velocities
+------------------------
 
-The LB fluid can be used to thermalize particles, while also including their hydrodynamic interactions.
-The LB thermostat expects an instance of either :class:`espressomd.lb.LBFluid` or :class:`espressomd.lb.LBFluidGPU`.
-Temperature is set via the ``kT`` argument of the LB fluid. Furthermore a seed has to be given for the
-thermalization of the particle coupling. The magnitude of the fricitional coupling can be adjusted by
-the parameter ``gamma``.
-To enable the LB thermostat, use::
+To get interpolated velocity values between lattice nodes, the function::
 
-    sys.thermostat.set_lb(LB_fluid=lbf, seed=123, gamma=1.5)
+    lb.get_interpolated_velocity(pos = [1.1,1.2,1.3])
+    
+with a single position  ``pos`` as an argument can be used. 
+For the GPU fluid :class:`espressomd.lb.LBFluidGPU`
+also :py:attr:`espressomd.lb.LBFluidGPU.get_interpolated_fluid_velocity_at_positions()`
+is available, which expects a numpy array of positions as an argument.
```

Comment:
could you please also mention the issue of interpolating velocities near boundaries?